drivers/bt_ll_nrfxlib/flash/flash_ll_nrfxlib.c

```diff
@@ -366,5 +366,6 @@ static int nrf_btctrl_flash_init(struct device *dev)
 	return 0;
 }
 
-DEVICE_INIT(nrf_btctrl_flash, DT_FLASH_DEV_NAME, nrf_btctrl_flash_init,
-	    NULL, NULL, POST_KERNEL, CONFIG_KERNEL_INIT_PRIORITY_DEVICE);
+DEVICE_INIT(nrf_btctrl_flash, DT_LABEL(SOC_NV_FLASH_NODE),
```

Comment:
@mbolivar-nordic Reported by @MarekPieta this is not working properly. He said `DT_LABEL(DT_PARENT(SOC_NV_FLASH_NODE))` seems more correct.
Is this correct, wouldn't the original `DT_INST(0, soc_nv_flash)` be more correct?
Was perhaps the original change correct, to follow the pattern in `soc_flash_nrf.c`?